Cross linking for Forms subjects (#1762)
PrestaShop/docs: development/architecture/migration-guide/forms/CRUD-forms.md

# CRUD Forms

{{% notice tip %}}
If you are looking to customize an identifiable object with a module, please have a look at [Grid and identifiable object form hooks usage example]({{<relref "/8/modules/sample-modules/grid-and-identifiable-object-form-hooks-usage">}}).
{{% /notice %}}

{{% notice tip %}}
If you are looking for the Form component documentation, please have a look at [Forms with Symfony]({{<relref "/8/development/components/form">}}).
{{% /notice %}}

{{% notice tip %}}

> This guide is intented for core developers looking to understand how PrestaShop now handles Forms in the core.

Reviewer: ```suggestion
This guide is intended for core developers and contributors who want to understand how PrestaShop handles forms in the core.
```
